bazel/_gevent_test_main.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import grpc
+import unittest
+import sys
+import os
+import pkgutil
+
+import zope
+from zope.event import subscribers
```

Comment:
Why do we need `zope`? It looks like another WSGI server.

Reply:
Good catch. We need `zope` as a transitive dependency, since `gevent` requires it, but we don't need this import here. I accidentally left this in from a debugging session. Removed.